src/libcore/ptr.rs

```diff
@@ -340,21 +600,31 @@ pub unsafe fn read_unaligned<T>(src: *const T) -> T {
 /// Overwrites a memory location with the given value without reading or
 /// dropping the old value.
 ///
-/// # Safety
-///
-/// This operation is marked unsafe because it accepts a raw pointer.
-///
-/// It does not drop the contents of `dst`. This is safe, but it could leak
+/// `write` does not drop the contents of `dst`. This is safe, but it could leak
 /// allocations or resources, so care must be taken not to overwrite an object
```

Comment:
I think this should also be more clear in the use of "must" (same applies to `write_unaligned`).

Reply:
I will replace it by "should" then.